plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -138,12 +138,14 @@ export const MembersListCard = (props: {
   memberDisplayTitle?: string;
   pageSize?: number;
   showAggregateMembersToggle?: boolean;
+  relationship?: string;
```

Comment:
Ooh so close, the perfect name for this one would be `relationType` because that's really what we call them, but it does indeed collide with the one below. That second one is badly named though. Could we do like

```ts
relationType?: string;
/**
 * @deprecated Please use `relationAggregation` instead
 */
relationsType?: EntityRelationAggregation;
relationAggregation?: EntityRelationAggregation;
```

and then the code does `const aggregation = options.relationAggregation ?? options.relationsType ?? 'direct'`? :) 🙏 just to get the interface nice while we're here

Reply:
Yes, I was disappointed to see that other name was (effectively) already taken :) I'll do as you suggest here

Reply:
@freben what should I do with the other usages of this same property name (`relationsType`) over in OwnershipCard? Should I deprecate those and provide a new property name there as well?